notify/src/windows.rs

```diff
@@ -138,8 +138,8 @@ impl ReadDirectoryChangesServer {
 
             unsafe {
                 // wait with alertable flag so that the completion routine fires
-                let waitres = WaitForSingleObjectEx(self.wakeup_sem, 100, 1);
-                if waitres == WAIT_OBJECT_0 {
+                let waiters = WaitForSingleObjectEx(self.wakeup_sem, 100, 1);
```

Comment:
that's intentional

Reply:
Whoops, I'd meant to remove those. Sorry about that! Removed these now.